guides-markdown/wallets.md

```diff
@@ -0,0 +1,357 @@
+	     accountIndex: 0,
+	     receiveDepth: 1,
+	     changeDepth: 1,
+	     nestedDepth: 0,
+	     lookahead: 10,
+	     address: <Address: type=pubkeyhash version=-1 str=mhNHETXFKDk7ZpGg3iEZb7guWZ2fbCuFjv>,
+	     nestedAddress: null,
+	     accountKey: 'tpubDDZ1r85SUsur87eW6uCrWancnCVHSLf5YcXzudCF6qBUQguR8upC6pgSuzxahDkf75SQ4LJ3R4x5NvfgQPmNjxhg2pcHzBCKcG2fBUQJ5U5', //the extended public key that can be used to generate receiving addresses for this account
+	     keys: [] } 
+ 	}*/
+```
+
+#### Creating Accounts and Receiving Addresses
+
+```javascript
+	const account = await wallet.getAccount('default');
+
+	//now we have a wallet and default account, let's get our first receiving public key and address
+	const key0 = account.deriveReceive(0);
+	console.log(key0);
+	/*{ network: 'testnet',
+	  wid: 2,
+	  id: 'WLTdx4aYEPmmrQiYNwPop4nLtbpTEdJYwrN4',
+	  name: 'default',
+	  account: 0,
+	  branch: 0,
+	  index: 100,
+	  witness: false,
+	  nested: false,
+	  publicKey: '02f4f200cb9391f8bbcc0a35e1f654b9b993b214a04ae7efd0313f4d4bf3d95745',
+	  script: null,
+	  program: null,
+	  type: 'pubkeyhash',
+	  address: 'mjVdQqQYWBpE6YzKyMRd96LxCMoJyeTX2i'
+	}*/
+
+	const key100 = account.deriveReceive(100); //we can also get the hundredth key in the heirarchy
+	console.log(key100.getAddress('string')); //to save your screen space, let's get just the address in string format
+	//mjVdQqQYWBpE6YzKyMRd96LxCMoJyeTX2i
+```
+
+We can create a second account and name it after a customer.
+
+```javascript
+	//ok that account is boring, let's create another one for hypothetical customer John Doe
+	const jdAccount = await wallet.createAccount({name: 'john_doe'});
+	console.log(jdAccount);
+	/*{ 
+	  wid: 2,
+	  name: '1',
+	  network: <Network: testnet>,
+	  initialized: true,
+	  witness: false,
+	  watchOnly: false,
+	  type: 'pubkeyhash',
+	  m: 1,
+	  n: 1,
+	  accountIndex: 1,
+	  receiveDepth: 1,
+	  changeDepth: 1,
+	  nestedDepth: 0,
+	  lookahead: 10,
+	  address: <Address: type=pubkeyhash version=-1 str=muCSbWC6z1tAr2i1M5BKPWEZ8zapzcKfKh>,
+	  nestedAddress: null,
+	  accountKey: 'tpubDDZ1r85SUsur9txJF5ziLRD6757E1Q7x6VLfPby4YKqAdNwgmrkXBNDzMowxYJVoAizd7CCLHELY5X2HYzh6YurbH9vMyQJN
+	T92n87z22yX',
+	  keys: [] 
+	}*/
+
+	//Mr. Doe wants to make 10 deposits, lets get him a unique address for each one
+	const depositAddressesToPrint = 10;
+	for(var i=0; i<depositAddressesToPrint; i++) {
+		console.log(jdAccount.deriveReceive(i).getAddress('string'));
+	}
+	/*
+	muCSbWC6z1tAr2i1M5BKPWEZ8zapzcKfKh
+	n4TWmQyPQ8mAr2oEfbBzKF8Dw6LXXBidYJ
+	ms1jRu71BvEsJ4K3dMFrKFUnB8axZQdmSq
+	mn8xFgB68RjWGdKPVb8Up4P4v5MyqPoEQj
+	mzfXXKRXJBjTGRFrrg4wm1XPsdU9TLoN6T
+	n36TShvFCDaWgCMHHszUGiczA7Tcru4AQp
+	mnZBaquULuUhtwwxKTuxWNF7ZDMDNxScJd
+	mfpwjevu3FY4ZsWPd31J9oBH1qrxZLF1tH
+	mhTrDspHReXThUmMeo8dVJHqhkyHG8VPZ1
+	mxKo27kJpNazq9Q3cQ7458k2S2vcQar9Pd
+	*/
+```
+
+#### Getting Private Keys
+
+```javascript
+	//the keys above are only good for receiving bitcoins, not spending them
```

Comment:
Nice! I like this section a lot. I feel like these types of examples will really help people understand what's going on under the hood.